syne_tune/optimizer/schedulers/searchers/bayesopt/models/sklearn_predictor.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+from typing import Optional, List, Dict
+
+import numpy as np
+
+from syne_tune.optimizer.schedulers.searchers.bayesopt.datatypes.tuning_job_state import (
+    TuningJobState,
+)
+from syne_tune.optimizer.schedulers.searchers.bayesopt.models.model_base import (
+    BasePredictor,
+)
+from syne_tune.optimizer.schedulers.searchers.bayesopt.sklearn.predictor import (
+    SklearnPredictor,
+)
+
+
+class SklearnPredictorWrapper(BasePredictor):
+    """
+    Wrapper class for the sklearn estimators to be used with ContributedEstimatorWrapper
+    """
+
+    def __init__(
+        self,
+        contributed_predictor: SklearnPredictor,
+        state: TuningJobState,
+        active_metric: Optional[str] = None,
+    ):
+        super().__init__(state, active_metric)
+        self.contributed_predictor = contributed_predictor
+
+    def predict(self, inputs: np.ndarray) -> List[Dict[str, np.ndarray]]:
+        """
+        Returns signals which are statistics of the predictive distribution at
+        input points ``inputs``. By default:
+
+        * "mean": Predictive means.
+        - "std": Predictive stddevs, shape ``(n,)``
```

Comment:
* instead of -

Reply:
But you could also drop this comment, because it is kind of just what the superclass has.

Reply:
Good point, I wated to change it for both but dropped it. Adjusted this and the base class now.